### ISSUE TITLE
ie9 `object is null or undefined`

### ISSUE THREAD

**Issue by amitport** [systemjs/systemjs#916]
using ie9 with `browser mode: IE9` and `Document Mode: IE9 standards`

loading system js `0.19.3`
```
SCRIPT5007: Unable to get value of the property 'script': object is null or undefined 
system.src.js, line 2505 character 17
```

**Comment by guybedford** [comment]
Did you include the `system-polyfills.js` file from https://github.com/systemjs/systemjs/tree/master/dist?

**Comment by amitport** [comment]
it was added automatically when loading in IE

**Comment by guybedford** [comment]
Try upgrading to 0.19.5.

**Comment by amitport** [comment]
fixed in 0.19.5

thanks!
